xunit.runner.visualstudio.testadapter/VsTestRunner.cs

```diff
@@ -363,9 +369,7 @@ void RunTests(IRunContext runContext, IFrameworkHandle frameworkHandle, LoggerHe
             var assemblyFileName = runInfo.AssemblyFileName;
             var assemblyDisplayName = Path.GetFileNameWithoutExtension(assemblyFileName);
             var shadowCopy = assembly.Configuration.ShadowCopyOrDefault;
-            var appDomain = assembly.Configuration.AppDomain ?? AppDomainDefaultBehavior;
-            if (DisableAppDomainRequestedInRunContext(runContext.RunSettings.SettingsXml))
-                appDomain = AppDomainSupport.Denied;
+            var appDomain = !RunSettingsHelper.DisableAppDomain ? AppDomainSupport.Denied : (assembly.Configuration.AppDomain ?? AppDomainDefaultBehavior);
```

Comment:
This is backwards.... currently you are saying, if `DisableAppDomain` is `false`, set `appDomain` to `AppDomainSupport.Denied`.